### ISSUE TITLE
Allow access to the annotation object using a forward ref

### ISSUE THREAD

**Issue by Nicolapps** [Nicolapps/mapkit-react#9]

**Comment by nikischin** [comment]
I agree, this could be possibly beneficial, though I would like to understand your need behind this. I was wondering, if it could be also beneficial providing the annotation reference on the events (additionally or instead) as some forward ref implementations in my experience can end up being quite an ugly solution hah

**Comment by Nicolapps** [comment]
I don’t have a need for it in my own projects, I created this issue more as something that could possibly be useful in the future. One example could be to access [`element`](https://developer.apple.com/documentation/mapkitjs/annotation/2973827-element), but I’m not sure what this could really be used for.

Since the MapKit JS APIs are rarely used, and should be avoided when possible, I would prefer to only expose them as refs. The ref system also makes the API usage more flexible, as it allows the users to pass the ref to other components or use them in event handlers of other elements.
